src/index.html

```diff
@@ -5,12 +5,12 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Mesto</title>
-  <link rel="stylesheet" href="./pages/index.css">
+  <link rel="stylesheet" href="<%=htmlWebpackPlugin.files.css %>">
 </head>
 <body>
   <div class="root">
     <header class="header root__section">
-      <img src="./images/logo.svg" alt="mesto logo" class="logo">
+      <img src="<%=require('./images/logo.svg')%>" alt="mesto logo" class="logo">
```

Comment:
Нужно исправить: изображение не отображается.
По-умолчанию `webpack` грузит все ресурсы как JavaScript код. Для того, чтобы после загрузки изображения ссылка на него была вставлена в HTML нужно указывать использовать свойство `.default`:
```suggestion
      <img src="<%=require('./images/logo.svg').default%>" alt="mesto logo" class="logo">
```